docs/en/stack/ml/df-analytics/ml-dfa-regression.asciidoc

```diff
@@ -24,7 +24,12 @@ the field you want to predict.
 //tag::regression-algorithms[]
 {regression-cap} uses an ensemble learning technique called extreme gradient 
 boost (XGboost) which combines decision trees with gradient boosting 
-methodologies.
+methodologies. XGboost trains a sequence of decision trees and every decision 
+tree learns from the mistakes of the previous one. Every tree is an iteration of 
+the last one, hence it improves the decision made by the previous tree. The 
+regression algorithm is optimized for a 
```

Comment:
Same as the text for classification.

Also, the regression algorithm **optimizes for** MSE.